src/test/java/uk/gov/justice/hmpps/casenotes/config/LocalStackConfig.kt

```diff
@@ -21,7 +21,6 @@ open class LocalStackConfig {
     log.info("Starting embedded localstack...")
     val localStackContainer: LocalStackContainer = LocalStackContainer()
         .withServices(LocalStackContainer.Service.SQS)
-        .withEnv("HOSTNAME_EXTERNAL", "localhost")
```

Comment:
not needed as default according to https://github.com/localstack/localstack#configurations is `localhost` anyway